src/modules/catalog.js

```diff
@@ -130,189 +131,35 @@ class Catalog {
-   * @param {number} [parameters.suggested_score] - A number between 1 and 100 million that will influence the item's initial ranking relative to other item scores (the higher the score, the higher in the list of suggestions the item will appear)
-   * @param {string[]} [parameters.keywords] - An array of keywords for this item. Keywords are useful if you want a product name to appear when a user enters a searchterm that isn't in the product name itself
-   * @param {string} [parameters.url] - A URL to directly send the user after selecting the item
-   * @param {string} [parameters.image_url] - A URL that points to an image you'd like displayed next to some item (only applicable when URL is supplied)
-   * @param {string} [parameters.description] - A description for some item (only applicable when URL is supplied)
-   * @param {string} [parameters.id] - An arbitrary ID you would like associated with this item. You can use this field to store your own IDs of the items to more easily access them in other API calls
-   * @param {object} [parameters.facets] - Key/value pairs that can be associated with an item and used to filter them during a search. You can associate multiple values with the same key, by making values a list. Facets can be used as filters in search, autosuggest, and browse requests
-   * @param {object} [parameters.metadata] - You can associate schema-less data with items by passing in an object of keys and values. To configure search and display of this data reach out to support@constructor.io
-   * @param {string[]} [parameters.group_ids] - You can associate each item with one or more groups (i.e. categories). To set up a group hierarchy please contact support@constructor.io. group_ids can be used as filters in search, autosuggest, and browse requests
-   * @param {object[]} [parameters.variations] - List of this item's variations
+   * @param {object[]} parameters.items - A list of items with the same attributes as defined in the `addItem` resource
    * @param {object} [networkParameters] - Parameters relevant to the network request
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @returns {Promise}
-   * @see https://docs.constructor.io/rest_api/items/modify_an_item
+   * @see https://docs.constructor.io/rest_api/items/items#create-or-replace-items
    */
-  modifyItem(parameters = {}, networkParameters = {}) {
+  createOrReplaceItems(parameters = {}, networkParameters = {}) {
     let requestUrl;
     const fetch = (this.options && this.options.fetch) || nodeFetch;
     const controller = new AbortController();
     const { signal } = controller;
+    const { section, key, force = false, notification_email, ...rest } = parameters;
+    const additionalQueryParams = {
+      section: section || 'Products',
+      key,
```

Comment:
I think the key should be coming from `this.options` and `createCatalogUrl` handles that. Do we need it here?